client/src/components/pages/Home/Messages/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import MessageCard from '../../../common/MessageCard';
+import './index.css';
+
+const Messages = ({ messages }) => {
```

Comment:
personally I would just put this into a function of the component where it is used, wouldn't add a separate component, doesn't add much. But up to you. But if it's a separate component you should define prop types

Reply:
We wanted to have it in a separate component